website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -1,193 +1,236 @@
+| [**updatedRows**](#updatedrows)              | Binding     | Contains all the data of the edited table rows.                                                    | `{{Table1.updatedRows}}`       |
+| [**updatedRowIndices**](#updatedrowindices)        | Binding     | Contains an array of indices of the table rows that have been edited.                                                        | `{{Table1.updatedRowIndices}}` |
+| [**updatedRow**](#updatedrow)               | Binding     | Contains the all the data of the row that was recently updated. | `{{Table1.updatedRow}}`|
+| [**Allow adding a row**](#allow-adding-a-row) | Widget | Toggles a button in the table which allows users to submit new rows of data. Only columns marked as **Editable** can accept user input. Use code or a query in the **onSave** event to update the source of the table's data and reflect the user's changes. |
+| [**Default Values**](#default-values) | Widget | The values to automatically populate the new row with when a user begins creating a new row. Expects an object with the same keys as the columns in the existing table data. |
+| [**isAddRowInProgress**]( #isaddrowinprogress) | Binding | Indicates whether a new row is currently being added by the user. | `Table1.isAddRowInProgress` |
+| [**newRow**](#newrow) | Binding | This variable contains a reference to the new row object added by the user. | `Table1.newRow` |
+| [**isNewRow**](#isnewrow) | Validation | When writing code for validation properties, this variable indicates whether the cells being validated are part of a new or an existing row. | `isNewRow` |
+
+## Events
+
+These event handlers can be used to run queries, JS code, or other [supported actions](/reference/appsmith-framework/widget-actions/) when the event is triggered.
+
+| Events        | Description                                                                                                            |
+| ------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| **onSubmit**  | Triggered when the user moves away from editing a cell by pressing the Enter key or clicking outside of the cell.      |
+| **onSave**    | Triggered when the user clicks the save button for a new or existing row.                                                |
+| **onDiscard** | Triggered when the user clicks the discard button for a new or existing row.                                             |
+
+---
+
 ## Editing cells
 
-The column list on the Table property pane has a checkbox to make that column editable.
+This feature enables users to make changes directly in the table cell. Currently, only a few column types support inline editing - **Text**, **Number**, **Switch**, and **Checkbox**. 
```

Comment:
We also have select column type

Reply:
Also please add a note that custom columns don't support editing as of now

Reply:
@spark-c I think you added this in the other place where this text occurs. Please make the changes here. and I think we can remove the second time it is repeated. 
<img width="953" alt="Screenshot 2022-12-12 at 18 17 03" src="https://user-images.githubusercontent.com/32227135/207048895-310e6f65-f21f-4a72-a2ff-59f635b65afb.png">